crates/egui/src/widgets/progress_bar.rs

```diff
@@ -15,6 +15,7 @@ pub struct ProgressBar {
     desired_height: Option<f32>,
     text: Option<ProgressBarText>,
     fill: Option<Color32>,
```

Comment:
What about calling the current fill `indicator_color` this could make it more clear?

Reply:
The reason for not doing something like this was because that would be a breaking change to the current API, so I opted for a less disruptive name choice. If the maintainers of egui are ok with a breaking change to the `ProgressBar` API, I think using names like the ones you suggest would be nice as well :)

Reply:
I think this would make it more clear for everyone!
@emilk what do you think?